active/NavigationView/BottomItems/BottomMenuItems.md

```diff
@@ -0,0 +1,155 @@
+
+# Background
+Today, [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) provides a place in its [Pane](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) to show any number of items.
+The primary purpose of these items is to be a list of navigation links within the app that hosts NavigationView, likely as a root element.
+This navigation list is top aligned when NavigationView is shown in one of its left [display modes](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes), and left aligned when NavigationView is in its top [display mode](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes).
+NavigationView provides a built-in [Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3), which is always positioned at the bottom or right of the Pane, and a [PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) area that positions content just before (either above or to the left) of the Settings item.
+Although an app developer can place any content into PaneFooter, including other NavigationViewItems, this content will not participate in the selection model and animations of the primary navigation list and Settings item.
+
+_Open question:_ Should PaneFooter be deprecated because this new API is a direct replacement?
```

Comment:
What happens if you use the old API?

Reply:
It's my understanding that if you add items to PaneFooter, they align to the bottom of the NavigationView but don't take part in the selection model - they cant be selected, and they aren't "counted" as items in the nav view, i.e. they don't have indices when navigating via keyboard. They do not raise ItemInvoked events when clicked on.

@ojhad can correct me if I'm wrong here.

Reply:
I just wanted to make sure that if you continue to use the footer that it won't regress with this change.

Reply:
That is correct. All `PaneFooter` does is display whatever gets passed into the property with no inherent behavioral expectations from NavigationView.

Functionality wise, I don't see this update regressing any existing implementation. However, we may need to think about the placement of the `PaneFooter` content in the pane relative to the footer list.